dts/bindings/sensor/adi,adltc2990.yaml

```diff
@@ -9,22 +9,52 @@ compatible: "adi,adltc2990"
 include: [sensor-device.yaml, i2c-device.yaml]
 
 properties:
-  mode_v1_v2:
+  temperature_format:
```

Comment:
Please squash the binding changes into one commit so you don't introduce a new binding in one commit and immediately change them in the next commit.

Reply:
my bad sorry